### PR TITLE
Jaeger: Remove advanced options toggle & show advanced options by default

### PR DESCRIPTION
**Pull request by CrypticSignal**
**What this PR does / why we need it**:
This PR removes the advanced options toggle when using Jaeger's search feature, and instead shows all options by default. This has two benefits:
- Users can more quickly see all the parameters that they can specify in a query.
- These options are shown by default when using Tempo, and therefore this change makes the user interfaces of these two data sources more consistent.

In addition, a couple of other improvements have been made:
- The `labelWidth` of the "Max Duration" and "Limit" fields has been reduced to 14, the same width as the other fields, giving the search form a more coherent style.
- The tooltip for the "Limit" field contained a typo - this has been fixed.

### CODE REVIEW

File: public/app/plugins/datasource/jaeger/components/AdvancedOptions.tsx [hunk 1]
```diff
@@ -1,131 +0,0 @@
-import { css } from '@emotion/css';
-import React, { useState } from 'react';
-import { CSSTransition } from 'react-transition-group';
-
-import { GrafanaTheme } from '@grafana/data';
-import { Icon, InlineField, InlineFieldRow, InlineLabel, Input, useStyles } from '@grafana/ui';
-
-import { JaegerQuery } from '../types';
-
-const durationPlaceholder = 'e.g. 1.2s, 100ms, 500us';
```
Comment (aocenas) [inline public/app/plugins/datasource/jaeger/components/AdvancedOptions.tsx:10]: Nitpick: We probably could have kept this a constant as it is used on 2 separate places.

**Review by aocenas — APPROVED** [review]
Looks good :+1:

File: public/app/plugins/datasource/jaeger/components/AdvancedOptions.tsx (hunk 1 above)
Comment (CrypticSignal) [reply public/app/plugins/datasource/jaeger/components/AdvancedOptions.tsx:10]: I agree, I will add this constant in `SearchForm.tsx`. I'm debating whether to add it just under the imports or just above the `SearchForm` component. Thoughts?

Comment (aocenas) [reply public/app/plugins/datasource/jaeger/components/AdvancedOptions.tsx:10]: honestly don't think it matters that much probably under imports same way as it was before is fine :)

Comment (CrypticSignal) [reply public/app/plugins/datasource/jaeger/components/AdvancedOptions.tsx:10]: Added in commit https://github.com/grafana/grafana/pull/53774/commits/ccf37b50a6b3acc271c7cbf08ac86acb24868052.

**Pull request merged**